src/main/java/use_cases/set_availability/SetAvailabilityController.java

```diff
@@ -0,0 +1,33 @@
+package use_cases.set_availability;
+
+/**
+ * Class responsible for initializing the first-time setting of availability;
+ * takes input of set of LocalTime values & gives it to SetAvailability interactor.
+ *
+ * @author od-obas1187
+ */
+public class SetAvailabilityController {
+    SetAvailabilityUseCase userAvailabilityInteractor;
```

Comment:
Clean architecture dictates that the Controller and the interactor should be separated by the InputBoundary; furthermore, you have yet to provide the access modifier and "final" modifier. I recommend adjusting this data attribute accordingly.